src/test_utils.rs

```diff
@@ -754,17 +755,43 @@ impl TestContext {
     }
 
     /// Returns the [`Contact`] for the other [`TestContext`], creating it if necessary.
+    ///
+    /// This function imports a vCard, so will transfer the public key
+    /// as a side effect.
     pub async fn add_or_lookup_contact(&self, other: &TestContext) -> Contact {
         let contact_id = self.add_or_lookup_contact_id(other).await;
         Contact::get_by_id(&self.ctx, contact_id).await.unwrap()
     }
 
-    /// Returns 1:1 [`Chat`] with another account. Panics if it doesn't exist.
+    /// Returns the [`Contact`] for the other [`TestContext`], creating it if necessary.
+    ///
+    /// If the contact does not exist yet, a new contact will be created
+    /// with the correct fingerprint, but without the public key.
+    pub async fn add_or_lookup_pgp_contact(&self, other: &TestContext) -> Contact {
```

Comment:
Since the difference to `add_or_lookup_contact()` is that the public key isn't transferred, would be nice to rename this to `add_or_lookup_contact_without_key()`.

Reply:
@link2xt agrees